Dockerfile

```diff
@@ -1,6 +1,7 @@
-FROM pytorch/pytorch:1.4-cuda10.1-cudnn7-runtime
+FROM pytorch/pytorch:1.6-cuda10.1-cudnn7-runtime
 
 RUN apt-get clean && apt-get -y update && apt-get install -y locales && locale-gen en_US.UTF-8
+
 ENV LC_CTYPE en_US.UTF-8
 ENV LANG en_US.UTF-8
 
```

Comment:
Add build-essential to the apt-get install list on line 14 (Github wont let me comment this there bc that line is unchanged)